### ISSUE TITLE
Gyro Emulation not working for reWASD.

### ISSUE THREAD

**Issue by MaestroLiendre** [LizardByte/Sunshine#1733]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Hey there! So I finally tried the beta with the gyro and works amazing, I've tried with my Logitech G Cloud and my Poco F3 + Gamesir x2 Pro and they both respond, but for some reason reWASD (a remapping software) seems to not be able to pick up the gyro inputs.

So I leave a video showing the controller where the gyro it's mapped to the mouse, so moving the gyro should move the mouse, but it doesn't, Everything else it's detected, just gyro it's not. Next to it is VSCview which it's actually showing that the gyro it's being picked up (and very precisely actually).

I don't know if gyro does send some "virtual signal" that reWASD it's not able to pick up, but VSCview does. It would be amazing if that could be fixed, as reWASD opens a world of possibilities with remapping.

I think it has something to with reWASD not being able to read the gyro inputs from vigembus, but no idea TBH.

Thanks a lot!

Link to the github issue I submitted in case is of any help: https://github.com/moonlight-stream/moonlight-android/issues/1258

### Additional Context

https://github.com/LizardByte/Sunshine/assets/147051719/086b5820-0c1a-412a-b5fa-1bf35763f739

### Host Operating System

Windows

### Operating System Version

Windows 11 Pro 22H2

### Architecture

64 bit

### Sunshine commit or version

Version 0.20.0.da04f97a8ce663769504ef2e41352cf13b65ecd9

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

3080Ti

### GPU Driver/Mesa Version

537.42

### Config

```Shell
resolutions = [
    1280x720,
    1920x1080,
    3840x2160,
    2160x1440
]
upnp = enabled
sunshine_name = Zalman Z9 Neo
fps = [30,60,90,120]
nvenc_preset = 4
hevc_mode = 0
encoder = nvenc
min_log_level = 1
```

### Relevant log output

```shell
I don't have a recent log where the issue is present.
```

**Comment by cgutman** [comment]
I have a fix in #1732. Please try out this build: https://github.com/LizardByte/Sunshine/suites/17054467852/artifacts/974076637

**Comment by MaestroLiendre** [comment]
> I have a fix in #1732. Please try out this build: https://github.com/LizardByte/Sunshine/suites/17054467852/artifacts/974076637

I'm at work for 8 more hours. I'll give you feedback as soon as I'm home.

Thanks.

**Comment by MaestroLiendre** [comment]
Wow!! That fixed it!! That was super quick! You can't imagine how greatful I am!!! You earned a new patreon well deserved!

**Comment by qiin2333** [comment]
> I have a fix in #1732. Please try out this build: https://github.com/LizardByte/Sunshine/suites/17054467852/artifacts/974076637

I tried this build with Logitech G Cloud, Still does not work.
